dagmc_stats/DagmcStats.py

```diff
@@ -22,7 +23,6 @@ def __init__(self, filename, populate=False):
         self._my_moab_core = core.Core()
         self._my_moab_core.load_file(filename)
         self.root_set = self._my_moab_core.get_root_set()
-        entity_types = [types.MBVERTEX, types.MBTRI, types.MBENTITYSET]
```

Comment:
If we make this `self.entity_types = ...` then it doesn't need to be passed the constructor above